plover/assets/main.json

```diff
@@ -97774,6 +97807,7 @@
 "SPHEU/TPHAR/-S": "seminars",
 "SPHEU/TROP/K-L": "semitropical",
 "SPHEUBG": "systemic",
+"SPHEUFD": "dismissed",
```

Comment:
`STKPHEUFD` would be better -- SMISD is a good brief but might confuse learners.